ingest.py

```diff
@@ -81,9 +73,14 @@ def ingest_avro(packet):
             gal_b=galactic.b.value,
             **packet['candidate']
             )
-        db.session.add(alert)
-        db.session.commit()
-        logger.info(alert.objectId)
+        try:
+            db.session.add(alert)
+            db.session.commit()
+            logger.info('Inserted object %s', alert.objectId)
```

Comment:
Now that we know candid is unique, let's use that in lieu of objectId where we can.

Reply:
Done.